src/webhooks/github/webhook_handlers/push.rs

```diff
@@ -58,7 +59,7 @@ pub fn handle_push_event(body: &web::Bytes) -> String {
         log::info!("Commit author: {}", commit.author.name);
 
         let commit_url = &commit.url;
-        let commit_message = &commit.message.trim_end();
+        let commit_message = encode_text(commit.message.trim_end());
         let commit_author_name = &commit.author.name;
```

Comment:
This should probably be encoded, too, just in case if someone uses funny symbols in their names.

Reply:
done